src/app.js

```diff
@@ -36,6 +33,9 @@ class App {
       });
     }
 
+    // errorHandler should be added as the last middleware to the app
+    app.use(errorHandler);
```

Comment:
Instead of having all middleware in the start function, lets have them in createExpressApp function and initialize all middleware within that function